src/frontend/src/optimizer/mod.rs

```diff
@@ -608,9 +608,6 @@ impl PlanRoot {
         let mut stream_plan = self.gen_stream_plan()?;
 
         match dml_flag {
-            // TODO: remove this branch after we deprecate the materialized source
-            DmlFlag::Disable => { /* do nothing */ }
-
             // NOTE(stonepage): we can not use this the plan's input append-only property here
             DmlFlag::All | DmlFlag::AppendOnly => {
```

Comment:
No need to match here?

Reply:
`DmlFlag::Disable` has been removed from the definition of this enum because it has no use.

Reply:
So that we can remove the arm `DmlFlag::All | DmlFlag::AppendOnly` as it's always true? 🥵